releases.json

```
@@ -114,4 +114,10 @@
     "extended_eol": false,
     "security_eol": "2028-02-28"
   }
```

please remember to add the comma between the objects :slightly_smiling_face: 

```suggestion
  },
```